File: src/main/java/com/my/admin/app/commerce/model/repository/OrderItemRepository.java
package com.my.admin.app.commerce.model.repository;

import com.black.food.manager.model.entity.OrderItem;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrderItemRepository extends JpaRepository<OrderItem, Long> {
}

File: src/main/kotlin/com/bk/reactive/app/commerce/admin/my/repository/OrderItemRepository.kt
package com.bk.reactive.app.commerce.admin.my.repository

import com.bk.reactive.app.commerce.admin.my.model.document.OrderItem
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository

@Repository
interface OrderItemRepository : ReactiveMongoRepository<OrderItem, String> {
}